build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id 'java'
     id 'jacoco'
-    id 'com.github.johnrengelman.shadow' version '7.1.2'
+    id 'com.github.johnrengelman.shadow' version '7.1.1'
```

Comment:
I had to change this in order to get the pipeline to build, as per the conversation in the dpr-developers slack channel